### PR TITLE
Makes amp-form uses ampstart

### PR DESCRIPTION
**Pull request by kul3r4**

### CODE REVIEW

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
Can you fix the html validation errors?

File: src/50_Samples_%26_Templates/Comment_Section.html [hunk 1]
```diff
@@ -23,9 +23,6 @@
     <link rel="canonical" href="<%host%>/samples_templates/comment_section/">
     <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
     <style amp-custom>
-    form {
-      padding-right: 0px;
-    }
     form.amp-form-submit-success [submit-success] {
         width: 100%;
         color: black;
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Comment_Section.html]: do we need this?

File: src/50_Samples_%26_Templates/Comment_Section.html [hunk 2]
```diff
@@ -50,12 +47,8 @@
         width: 100%;
     }
     .user-avatar {
-      margin-right: 16px;
       flex-shrink: 0;
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Comment_Section.html]: can we replace `flex-shrink: 0` with the `flex-none` basscss class?

File: src/50_Samples_%26_Templates/Comment_Section.html [hunk 3]
```diff
@@ -50,12 +47,8 @@
...
     }
-    .comments-container {
-      padding-left: 16px;
-    }
     .comment-user {
       display: flex;
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Comment_Section.html]: we can replace this with the `flex` class

File: src/50_Samples_%26_Templates/Comment_Section.html [hunk 4]
```diff
@@ -23,9 +23,6 @@
     <link rel="canonical" href="<%host%>/samples_templates/comment_section/">
     <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
     <style amp-custom>
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Comment_Section.html]: Two formatting issues in this sample:

1. large space below the comment list
2. new comment is rendered in the same line

screen shot 2017-02-28 at 06 22 03

File: src/50_Samples_%26_Templates/Housing.html [hunk 5]
```diff
@@ -117,21 +117,15 @@
     }
     .form-field-right {
       border: 1px solid #dfdfdf;
-      position: relative;
       background: white;
       width: 200px;
-      width: 179px;
     }
     .form-field-right input {
       border: none;
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Housing.html]: we can remove this rule - can't we?

File: src/50_Samples_%26_Templates/Housing.html [hunk 6]
```diff
@@ -117,21 +117,15 @@
...
     .form-field-left {
       width: 200px;
     }
-    .form-field {
-      display: flex;
-      margin-bottom: 16px;
-    }
     .description {
       padding: 16px;
       margin: 16px;
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Housing.html]: are the padding/margin still required in this rule?

File: src/50_Samples_%26_Templates/Poll.html [hunk 7]
```diff
@@ -114,25 +103,26 @@ <h4 id="header1">Flightless bird poll</h4>
         <ul>
           [[range $key, $value := .Questions]]
           <li>
+              <input id="[[$key]]" name="answer" value="[[$key]]" type="radio" on="change:poll1.submit">
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Poll.html]: This sample didn't work for me anymore. The result wasn't rendered.

File: src/50_Samples_%26_Templates/Product_Listing.html [hunk 8]
```diff
@@ -31,12 +31,6 @@
       background: #ECEFF1;
     }
     <%>css/product.css%>
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product_Listing.html]: the search input field is mis-aligned in the listing sample, works in the product sample.

File: src/50_Samples_%26_Templates/Poll.html (hunk 7 above)
Comment (kul3r4) [inline src/50_Samples_%26_Templates/Poll.html]: Works for me
screen shot 2017-02-28 at 9 44 52 am

**Comment by kul3r4** [comment]
Applied all changes, @sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
Looks great! Noticed a few minor alignment issues:

Can you align submit button with first radio button?
screen shot 2017-02-28 at 11 36 43

add margin below submit button (might make sense to check all form samples for this)
screen shot 2017-02-28 at 11 32 24
screen shot 2017-02-28 at 11 27 40

Add margin left to paragraphs
screen shot 2017-02-28 at 11 30 19
screen shot 2017-02-28 at 11 30 11

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by kul3r4** [comment]
@sebastianbenz you are right, just fixed it

**Comment by sebastianbenz** [comment]
Somehow this change seemed to made it worse:

screen shot 2017-02-28 at 14 02 57
screen shot 2017-02-28 at 14 02 46
screen shot 2017-02-28 at 14 02 40
screen shot 2017-02-28 at 14 02 02
screen shot 2017-02-28 at 14 01 58
screen shot 2017-02-28 at 14 01 53

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by lisaychuang** [comment]
Noted this sample for "Hotel Listing Page" is not visible on ampbyexample.com, not sure if a link needs to be added to the menu bar - sample?
alt text

**Pull request merged**